misc/config_tools/board_config/board_c.py

```diff
@@ -15,9 +15,9 @@ class RDT(enum.Enum):
     MBA = 2
 
 INCLUDE_HEADER = """
-#include <x86/board.h>
-#include <x86/vtd.h>
-#include <x86/msr.h>
+#include <asm/board.h>
+#include <asm/vtd.h>
+#include <asm/msr.h>
 #include <pci.h>
```

Comment:
Do you regenerate the files under  misc\config_tools\data\generic_board\generic_code\boards to reflect your changes of config_tools?